data/templates/snom.macros

```diff
@@ -34,6 +34,32 @@
 {% endmacro linekey_type_map %}
 
 
+#
+# softkey_type_map -- translate "softkey_type_"
+#
+{% macro softkey_type_map(softkey_type) %}
+{% if softkey_type == "" %}
+    {{- 'keyevent F_NONE' }}
+{% elseif softkey_type == "forward" %}
+    {{- 'keyevent F_REDIRECT' }}
+{% elseif softkey_type == "dnd" %}
+    {{- 'keyevent F_DND' }}
+{% elseif softkey_type == "recall" %}
+    {{- 'keyevent F_REDIAL' }}
+{% elseif softkey_type == "history" %}
+    {{- 'keyevent F_CALL_LIST' }}
+{% elseif softkey_type == "menu" %}
+    {{- 'keyevent F_SETTINGS' }}
+{% elseif softkey_type == "status" %}
+    {{- 'keyevent F_STATUS' }}
+{% elseif softkey_type == "ldap" %}
+    {{- 'keyevent F_DIRECTORY_SEARCH' }}
+{% else %}
+    {{- softkey_type }}
+{% endif %}
+{% endmacro softkey_type_map %}
```

Comment:
why haven't you used a map like this https://github.com/nethesis/tancredi/blob/9f1efcb850a6b6ed4ce637caf462297369f5ef67/data/templates/snom.macros#L97 ?

Reply:
Do you mean to use `{% macro language(language)%}` instead of `{% macro softkey_type_map(softkey_type) %}` in line line 40th ?
Or to use it in line 58th ?
`snom.macros` contains:
```
#
# Map ISO 639-1 codes to localized languages used by Snom
#
{% macro language(language)%}
{% set language_map = {
    "bs" : "Bosanski",
    "da" : "Dansk",
    "de" : "Deutsch",
    "cs" : "Čeština",
```
and I don't understand why you would like to use it in the mentioned line of code

Reply:
I mean use the map like in language macro

Reply:
NVM, was just an opinion. It's ok for me

Reply:
Ok, now I got it.
We have spoken about that and we have decided to leave the code as is, to avoid lost time.